ui/lib/kv/addon/components/page/secret/metadata/details.hbs

```diff
@@ -80,7 +109,7 @@
   <h2 class="title is-5 has-bottom-padding-s has-top-margin-l">
     Secret metadata
   </h2>
-  {{#if @canReadMetadata}}
+  {{#if @metadata}}
```

Comment:
if we have `@metadata` we `canReadMetadata` - the previous check caused accessibility failures for the `KvTooltipTimestamp`